js/api_call.js

```diff
@@ -1,21 +0,0 @@
-var apiCalls = {
-    vibrationApi: function () {
```

Comment:
This function vibrates the device, so vibrartionApi doesnot make sense. Name should be something similar to the work it performs.